src/main/kotlin/andrasferenczi/templater/VariableTemplateParam.kt

```diff
@@ -1,11 +1,13 @@
 package andrasferenczi.templater
 
 interface NamedVariableTemplateParam {
+    val isFinal: Boolean
```

Comment:
I don't really understand how it works, these are the questions that came up:

- so `isFinal` field is only needed to be able to tell if it should get the `required` keyword added in case of `public` fields (see question on this below)
- I already have a [`DeclarationModifier.kt`](https://github.com/andrasferenczi/dart-data-plugin/blob/master/src/main/kotlin/andrasferenczi/declaration/DeclarationModifier.kt) (I don't remember much of the code after almost 2 years) that already features a `final` modifier. How does this `isFinal` relate to that?
- I think there must be an another way of adding this `isFinal`, because the class is meant for variable names only and modifiers come elsewhere

Reply:
> I don't really understand how it works, these are the questions that came up:
> 
> * so `isFinal` field is only needed to be able to tell if it should get the `required` keyword added in case of `public` fields (see question on this below)
> * I already have a [`DeclarationModifier.kt`](https://github.com/andrasferenczi/dart-data-plugin/blob/master/src/main/kotlin/andrasferenczi/declaration/DeclarationModifier.kt) (I don't remember much of the code after almost 2 years) that already features a `final` modifier. How does this `isFinal` relate to that?
> * I think there must be an another way of adding this `isFinal`, because the class is meant for variable names only and modifiers come elsewhere

@andrasferenczi @mzdm Sorry, I'm not familiar with the code enough to answer this comment.